src/component/Label.tsx

```diff
@@ -498,11 +510,11 @@ const parseLabel = (label: unknown, viewBox: ViewBox) => {
     return <Label key="label-implicit" content={label} viewBox={viewBox} />;
   }
 
-  if (isFunction(label)) {
+  if (isValidElement(label) || typeof label === 'function') {
```

Comment:
this should just be `typeof label === 'function'`

We already handled if it is a valid element up above